### PR TITLE
add README_zh-CN.md

### PR DESCRIPTION
**Pull request by Forest-L**
Signed-off-by: Forest-L <lilin@yunify.com>

### CODE REVIEW

File: README_zh-CN.md [hunk 1]
```diff
@@ -0,0 +1,166 @@
+# KubeEye
+
+KubeEye旨在发现Kubernetes上的各种问题，比如应用配置错误（使用[Polaris](https://github.com/FairwindsOps/polaris)）、集群组件不健康和节点问题（使用[Node-Problem-Detector](https://github.com/kubernetes/node-problem-detector)）。除了预定义的规则，它还支持自定义规则。
```
Comment (FeynmanZhou) [inline README_zh-CN.md:3]: 英文与中文之间、数字与中英文间左右需各空一格

File: README_zh-CN.md [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+root:# ke diag -f rule.yaml --kubeconfig ~/.kube/config
+NAMESPACE     SEVERITY    NAME                      KIND         TIME                        MESSAGE
+default       Warning     nginx                     Deployment   2020-11-27T17:18:31+08:00   [imageFromUnauthorizedRegistry]
+kube-system   Warning     node-problem-detector     DaemonSet    2020-11-27T17:18:31+08:00   [livenessProbeMissing runAsPrivileged]
+kube-system   Warning     calico-node               DaemonSet    2020-11-27T17:18:31+08:00   [cpuLimitsMissing runAsPrivileged]
+kube-system   Warning     calico-kube-controllers   Deployment   2020-11-27T17:18:31+08:00   [cpuLimitsMissing livenessProbeMissing]
+kube-system   Warning     nodelocaldns              DaemonSet    2020-11-27T17:18:31+08:00   [runAsPrivileged cpuLimitsMissing]
+default       Warning     nginx                     Deployment   2020-11-27T17:18:31+08:00   [livenessProbeMissing cpuLimitsMissing]
+kube-system   Warning     coredns                   Deployment   2020-11-27T17:18:31+08:00   [cpuLimitsMissing]
+```
```
Comment (FeynmanZhou) [inline README_zh-CN.md:166]: 在末尾加上 Roadmap 吧

File: README_zh-CN.md (hunk 1 above)
Comment (LinuxSuRen) [reply README_zh-CN.md:3]: 可以出个规范，大家也方便了解

Comment (Forest-L) [reply README_zh-CN.md]: fixed it.

File: README_zh-CN.md (hunk 2 above)
Comment (Forest-L) [reply README_zh-CN.md:166]: fixed it.

File: README_zh-CN.md (hunk 1 above)
Comment (FeynmanZhou) [reply README_zh-CN.md]: 规范可以参考 [localization guide](https://github.com/kubesphere/website/blob/master/localization_style_guides/KubeSphere%20Localization%20Style%20Guide%20(for%20Simplified%20Chinese).md#kubesphere-localization-style-guide-for-simplified-chinese).

**Review by FeynmanZhou — APPROVED** [review]

**Pull request merged**